memory/src/test/java/org/opensearch/ml/memory/action/conversation/CreateInteractionRequestTests.java

```diff
@@ -104,10 +104,12 @@ public void testFromRestRequest() throws IOException {
             );
 
         RestRequest rrequest = new FakeRestRequest.Builder(NamedXContentRegistry.EMPTY)
-            .withParams(Map.of(ActionConstants.CONVERSATION_ID_FIELD, "cid"))
+            .withParams(Map.of(ActionConstants.MEMORY_ID, "cid"))
             .withContent(new BytesArray(gson.toJson(params)), MediaTypeRegistry.JSON)
             .build();
         CreateInteractionRequest request = CreateInteractionRequest.fromRestRequest(rrequest);
+        System.out.println(request.getConversationId());
```

Comment:
Remove `System.out.println`

Reply:
Good catch. Removed them all. These are leftover when debugging the Tests after refactoring.